tests/test_xmi.py

```diff
@@ -66,9 +67,9 @@ def test_views_are_parsed(small_xmi, small_typesystem_xml):
     <xmi:XMI xmlns:tcas="http:///uima/tcas.ecore" xmlns:xmi="http://www.omg.org/XMI" xmlns:cas="http:///uima/cas.ecore"
              xmlns:cassis="http:///cassis.ecore" xmi:version="2.0">
         <cas:NULL xmi:id="0"/>
-        <tcas:DocumentAnnotation xmi:id="8" sofa="1" begin="0" end="47" language="x-unspecified"/>
```

Comment:
Again, I wonder why the offsets changed here and in the following lines...?

Reply:
Wrong before, but we never checked them.